chemicalx/constants.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for ChemicalX."""
+
+__all__ = [
+    "TORCHDRUG_NODE_FEATURES",
+]
+
+#: The default number of node features on a molecule in torchdrug
+TORCHDRUG_NODE_FEATURES = 69
```

Comment:
We might need to fix the torchdrug version. What do you think @cthoyt ?

Reply:
Done in 2292b4a